container-core/src/main/resources/configdefinitions/logserver.def

```diff
@@ -0,0 +1,8 @@
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+namespace=container.core
+
+## Path to log directory
+logDirectory string default="/home/y/logs/vespa/logarchive/"
```

Comment:
Alternative is to make this a path under `VESPA_HOME`, then we can avoid `/home/y/`, but are limited to logs under `VESPA_HOME`

Reply:
Or maybe you could use a relative path for paths under VESPA_HOME as a convention?

You might want to change the name of the config, though, as it is a config for the handler and e.g. for host-admin it will have nothing to do with logserver

Reply:
Fixed, PTAL